src/giddyup.erl

```diff
@@ -27,6 +27,7 @@
 get_suite(Platform) ->
```

Comment:
Does it make sense for this module to be a gen_server?  It would be nice to remove the reliance on rt_config and pass the state information into it from the escript.  Also, it would remove potential blocking of the new pipeline if we have a slow connection to Giddyup.